mbtools/validate_mbz_html.py

```diff
@@ -0,0 +1,118 @@
+import argparse
+import json
+from pathlib import Path
+from . import utils
+
+STYLE_VIOLATION = "ERROR: Uses In-Line Styles"
+SOURCE_VIOLATION = "ERROR: Uses External Resource with Invalid Prefix"
+MOODLE_VIOLATION = "ERROR: References to Uploaded Files in Moodle DB"
+SCRIPT_VIOLATION = "ERROR: Uses In-Line Script Element"
+IFRAME_VIOLATION = "ERROR: Uses In-Line iFrame Element"
+
```

Comment:
I think we're currently missing `<a>` elements with unexpected `href` attributes. It's TBD how strict we want to be on this, but maybe we can tune the initial accept list based on errors we see from the thin slice / unit 1.

Reply:
Oh, yeah, it was sort of alluded to in item 7 from the description list but I didn't specifically call out the details. One might be a link to `desmos.com` like the following (since we want Demos activities to be embedded in some manner):

```html
<a href="https://teacher.desmos.com/activitybuilder/custom/58470c3efe07d5dd05d7f4a6" target="_blank">Algebraic Expressions and Word Phrases&nbsp;</a>
```

I'm not sure what types of links are currently in the content in general, so I expect this rule to be loud with false positives until we tune it some.